manifests/v3/conf/sites.pp

```diff
@@ -1,67 +0,0 @@
-# == Class: freeradius::conf::sites
-#
-# Manage the 'sites' section of radiusd.conf.
-#
-# You can only call this *once* within a node scope. If you try to call it more
-# than once, it will fail your manifest compilation due to conflicting
-# resources.
-#
-# See /etc/raddb/radiusd.conf.sample for additional information.
-#
-# == Parameters
-#
-# @param enable_default
-#  Boolean:
-#   Whether or not to enable the default site from the FreeRADIUS package.
-#  Default: true
-#
-# @param enable_inner_tunnel
-#  Boolean:
-#   Whether or not to enable the inner_tunnel site.
-#  Default: true
-#
-# == Authors
-#
-# * Kendall Moore <kmoore@keywcorp.com>
-#
-class freeradius::v3::conf::sites (
-  $enable_default      = true,
-  $enable_inner_tunnel = true
-){
-  include '::freeradius'
-
-  if $::operatingsystem in ['RedHat', 'CentOS'] {
-    if defined('$::radius_version') and ($::radius_version != 'unknown') {
-      if (versioncmp($::radius_version, '3') >= 0) {
-        file { '/etc/raddb/conf/sites.inc':
```

Comment:
I don't know if the user has need for them.  They can include them if they want to though.  They do not need them for ldap as far as I can tell.  They have LDAP in them because they can use LDAP if they are configured.